nym-connect-android/src/stories/AppFlow.stories.tsx

```diff
@@ -36,27 +36,27 @@ export const Mock: ComponentStory<typeof AppWindowFrame> = () => {
   ];
   const handleConnectClick = React.useCallback(() => {
     const oldStatus = context.connectionStatus;
-    if (oldStatus === ConnectionStatusKind.connected || oldStatus === ConnectionStatusKind.disconnected) {
+    if (oldStatus === 'connected' || oldStatus === 'disconnected') {
       setBusy(true);
 
       // eslint-disable-next-line default-case
       switch (oldStatus) {
-        case ConnectionStatusKind.disconnected:
+        case 'disconnected':
           context.setConnectionStatus(ConnectionStatusKind.connecting);
           break;
-        case ConnectionStatusKind.connected:
+        case 'connected':
           context.setConnectionStatus(ConnectionStatusKind.disconnecting);
           break;
       }
 
       setTimeout(() => {
         // eslint-disable-next-line default-case
         switch (oldStatus) {
-          case ConnectionStatusKind.disconnected:
+          case 'disconnected':
             context.setConnectedSince(DateTime.now());
             context.setConnectionStatus(ConnectionStatusKind.connected);
             break;
-          case ConnectionStatusKind.connected:
+          case 'connected':
             context.setConnectionStatus(ConnectionStatusKind.disconnected);
             break;
```

Comment:
These should still use the enum

Reply:
in `develop` version we're using the union type as well ->
https://github.com/nymtech/nym/blob/37187c79ccee8ab64fc48f1f92d9983f0da57e9e/nym-connect/src/stories/AppFlow.stories.tsx#L39